express-get/server.ts

```diff
@@ -0,0 +1,49 @@
+import express from 'express';
+import pg from 'pg';
+import { errorMiddleware } from './lib/index.js';
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/pagila',
+  ssl: {
+    // Allow non-SSL traffic to localhost
+    rejectUnauthorized: false,
+  },
+});
+
+const app = express();
+
+app.get('/api/films', async (req, res, next) => {
+  try {
+    const sql = `
+    select * from "films"
+    `;
+    const result = await db.query(sql);
+    const films = result.rows;
+    res.json(films);
+  } catch (err) {
+    next(err);
+  }
+});
+
+app.get('/api/films/:filmId', async (req, res, next) => {
+  try {
+    const sql = `
+    select "filmId" from "films"
+    `;
+    const result = await db.query(sql);
+    const films = result.rows;
+    res.json(films);
+
+    if (!Number.isInteger(+films)) {
```

Comment:
You need to check this _before_ you run the query. Otherwise, you might run an invalid query. You should always check the request values as the very first thing you do in your route handler function.